pkgs/desktops/gnome/core/totem/default.nix

```diff
@@ -91,9 +91,7 @@ stdenv.mkDerivation rec {
     adwaita-icon-theme
     gnome-desktop
     gsettings-desktop-schemas
-    # for plug-ins
     python3Packages.pygobject3
-    python3Packages.dbus-python
```

Comment:
Yeah in xplayer packaging (as a fork of old totem) documented this as for dbusservice plugin but this plugin is likely gone in new totem these days :shrug: : https://github.com/GNOME/totem/commit/166e95afd34e389121a5050cce677fb6ae90ce8a